src/Filters/response.jl

```diff
@@ -58,6 +60,25 @@ function phasez(filter::FilterCoefficients, w = range(0, stop=π, length=250))
 end
 
 
+"""
+    grpdelay(fliter, w = range(0, stop=π, length=250))
+
+Group delay of a digital 'filter' at normalized frequency
+or frequencies 'w' in radians/sample.
+"""
+function grpdelay(filter::FilterCoefficients, w = range(0, stop=π, length=250))
+    filter = convert(PolynomialRatio, filter)
+    b, a = coefb(filter), filter.a.coeffs # reversed a
+    c = conv(b, conj(a))
+    cr = range(0, stop=length(c)-1) .* c
+    ejw = exp.(-im * w)
```

Comment:
```suggestion
    ejw = exp.(-im .* w)
```
gets rid of a temporary array.